intro.qmd

```diff
@@ -149,21 +128,16 @@ They include reusable functions, documentation that describes how to use them, a
 
 ### R
 
-To download R, go to CRAN, the **c**omprehensive **R** **a**rchive **n**etwork.
-CRAN is composed of a set of mirror servers distributed around the world and is used to distribute R and R packages.
-Don't pick a mirror close to you; instead, use the cloud mirror, <https://cloud.r-project.org>, which automatically figures it out for you.
-
+To download R, go to CRAN, the **c**omprehensive **R** **a**rchive **n**etwork, <https://cloud.r-project.org>.
 A new major version of R comes out once a year, and there are 2-3 minor releases each year.
 It's a good idea to update regularly.
 Upgrading can be a bit of a hassle, especially for major versions requiring you to re-install all your packages, but putting it off only makes it worse.
-You'll need at least R 4.1.0 for this book.
+We recommend R 4.2.0 or later for this book.
 
 ### RStudio
 
-RStudio is an integrated development environment, or IDE, for R programming.
-Download and install it from <https://posit.co/download/rstudio-desktop/>.
-RStudio is updated a couple of times a year.
-When a new version is available, RStudio will let you know.
+RStudio is an integrated development environment, or IDE, for R programming, which you can download from <https://posit.co/download/rstudio-desktop/>.
+RStudio is updated a couple of times a year, but RStudio will automatically let you know when a new version is out so there's no need to check back.
```

Comment:
```suggestion
RStudio is updated a couple of times a year; it will automatically let you know when a new version is out so there's no need to check back.
```